### PR TITLE
Initial deploy of concept ingester

### PR DESCRIPTION
**Pull request by scott-ace-newton**

### CODE REVIEW

**Comment by tamas-molnar** [comment]
+1

File: concept-ingester-blue@.service [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ExecStartPre=-/bin/bash -c '/usr/bin/docker kill %p-%i > /dev/null 2>&1'
+ExecStartPre=-/bin/bash -c '/usr/bin/docker rm %p-%i > /dev/null 2>&1'
+ExecStartPre=/bin/bash -c 'docker history coco/concept-ingester:$DOCKER_APP_VERSION > /dev/null 2>&1 || docker pull coco/concept-ingester:$DOCKER_APP_VERSION'
+
+ExecStart=/bin/sh -c '\
+  export PORT=8080; \
+  export VULCAN_ADDR=http://%H:8080; \
+  export GROUP_ID=BlueConcepts; \
+  export TOPIC=Concept; \
+  export SERVICES=people-rw-neo4j-blue,organisations-rw-neo4j-blue,memberships-rw-neo4j-blue; \
```
Comment (tosan88) [inline concept-ingester-blue@.service]: Could we add other rw services, like `brands-rw-neo4j-blue`?

Comment (scott-ace-newton) [inline concept-ingester-blue@.service]: are you asking me to add them? or are you asking if it is possible?

Comment (tosan88) [inline concept-ingester-blue@.service]: I think we need brands too here, @nwrigh @BerniVarga am i right?

Comment (nwrigh) [inline concept-ingester-blue@.service]: Yes and roles-rw

**Comment by tosan88** [comment]
:+1:

**Pull request merged**